### DISCUSSION TITLE
Did you retire/archive all content related to SharePoint Server and CSOM? 🤔

### DISCUSSION THREAD

**Discussion by b-dasass** [SharePoint/sp-dev-docs#7556] [Q&A] [answered]
Am I being just so lame that I cannot find documentation related to `Microsoft.SharePoint.Client.ListCollection` for SharePoint Server 2016 & 2019 **OR** there is actually no official documentation available to read which is not about .NET Standard? 

Any and all help would be appreciated!

**Comment by patmill** [comment]
Do you mean this documentation?
https://docs.microsoft.com/en-us/sharepoint/dev/sp-add-ins/sharepoint-net-server-csom-jsom-and-rest-api-index
https://docs.microsoft.com/en-us/previous-versions/office/sharepoint-visio/jj246453(v=office.15)

**Reply by b-dasass** [reply]
I'm sorry if I wasn't clear. I'm not looking for `SP.ListCollection`. I'm looking for the .NET Framework based version of Client Side Object Model which can be found inside [Microsoft.SharePoint2016.CSOM](https://www.nuget.org/packages/Microsoft.SharePoint2016.CSOM/)

**Comment by patmill** [comment, marked as answer]
So for CSOM, there is this - https://docs.microsoft.com/en-us/previous-versions/office/sharepoint-csom/ee546609(v=office.15)

I'm checking to see if there is are updated docs somewhere.

**Reply by b-dasass** [reply]
Thank you!

I was trying to build some older - now archived - PnP sample for SharePoint 2013 but some methods were not in CSOM so it didn't want to build, but the docs said that they should be there...

So now I'm thinking that this `v=office.15` might be for `v=office.16` because the 2016 CSOM version had that method. 

I hope this make sense.

**Comment by patmill** [comment]
<tangent> It's unfortunate that the version and the year of the releases is close-but-not-exact.  O16 is not office 2016.  To complicate matters further, SharePoint went from having it's own version #s (SharePoint v3 was the SP2007 for example).  At some point, the SharePoint releases and the Office release #s merged, and started using the Office Release #s, which is how we wound up at version 15 and above.  </tangent>
